docs/adr/decisions/0003-item-timestamps-managed-internally.md

```diff
@@ -0,0 +1,50 @@
+---
+status: "proposed"
+---
+# Created and Modified Timestamps should be managed internally
+
+## Context and Problem Statement
+
+Flows (and Sources if [ADR 002](./0002-add-sources-to-api.md) is accepted) have fields indicating when they were created and last modified, however the API specification and schemas are not clear on whether these fields can be modified by a client, or whether they are exclusively managed by the server.
+
+See [schemas/flow-core.json](../../../api/schemas/flow-core.json) for definitions of those fields.
+They exist primarily to make it easier to find items of interest in the API and support engineering functions such as application development and debugging.
+
+## Considered Options
+
+* Option 1: Allow clients to change the dates when making a PUT request
+* Option 2: Ignore the date-time values in client requests
+
+## Decision Outcome
+
+Chosen option: Option 2 - Ignore the date-time values in client requests, because it allows users to trust the behaviour of the date-time values, which is important given their primary purpose is to support development and debugging.
+
+This limits the use of the `created` field to refer to when the Flow was originally captured, but that information should ideally live elsewhere in a MAM system, or in the basic case, could be represented using tags.
```

Comment:
Not sure I understand this, but I assume with option 2 that the `created` field refers to when the Flow was stored rather than originally captured?

Reply:
Yes - as flow-core.json puts it:
> The date-time the flow was created in a given context, e.g. in a store.

With hindsight I don't like this very much - I can't see why you'd try and represent when the original media was originally captured using `created` anyway, so I've taken it out. If the time at which the first sample occurred is important, then you start the Flow timeline from there I think.